docs/acquisition.rst

```diff
@@ -0,0 +1,167 @@
+Acquiring multi-echo data
+=========================
+
+Available multi-echo fMRI sequences
+-----------------------------------
+
+We have attempted to compile some basic multi-echo fMRI protocols in an `OSF project`_.
+These protocols should not be considered "canonical".
+If you would like to use one of them, please customize it for your own purposes
+and make sure to pilot it before using it in a study.
```

Comment:
@handwerkerd In fixing the OSF link you noticed, I also realized that we have a ME-fMRI sequences project on the OSF as well (https://osf.io/ebkrp/). I want to link to it (even though we don't actually have anything in there yet), but also want to make sure that the language reflects your past concerns that folks would treat any protocols we provide as "best practice" without critically evaluating and customizing them themselves.

What do you think of the above language?

Reply:
How about: "The parameter choices in these protocols run and seem reasonable, but the have not been optimized for a specific situation. They are a good starting point for someone designing a study, but should not be considered canonical. If you would like to use one of them, please customize it for your own purposes and make sure to run pilot scans to test your choices."

Reply:
Added. Thanks!